README.md

```diff
@@ -116,6 +116,15 @@ That's the entire API.
 
 <br>
 
+## Workspaces
+
+Support for member crates of a workspace is currently limited to crate  specifying the workspace path explicitly within the  `Cargo.toml` manifest, which would usually look something like this [(more info)](https://doc.rust-lang.org/cargo/reference/manifest.html#the-workspace--field-optional):
+
+```toml
+[package]
+workspace = ".."
```

Comment:
Please detect the workspace root from `cargo metadata` rather than requiring this. See the existing code in `crate::cargo::target_dir()`. Look for a key called "workspace_root".

Reply:
Cool, that's much better!

Fixed.